test/src/resource_test.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
This file structure is strange.
test directory contains only test file.

Reply:
src directory contains some implementation, but containing only test file is strange.

Reply:
Refer to https://github.com/okuda-seminar/client-go/pull/2#discussion_r1311178569